infra/README.md

```diff
@@ -11,18 +11,49 @@
 
 ## Deploying the CDK
 To deploy the CDK stacks, you must have the appropriate permissions to create
-the CloudFormation stack and associated resources in a given AWS account. You 
-will also need:
-- A personal access token for the [Prow bot account](https://github.com/kubernetes/test-infra/blob/master/prow/getting_started_deploy.md#github-bot-account)
-- An HMAC token used to validate Github webhooks
-  - This can be generated using `openssl rand -hex 20`
+the CloudFormation stack and associated resources in a given AWS account.
+
+You will also need to **manually** create and configure a GitHub app for Prow as [documented here](https://github.com/kubernetes-sigs/prow/blob/main/site/content/en/docs/Legacy%20Snapshot/prow/getting_started_deploy.md#github-app).
```

Comment:
This link 404s for me, I see a file in the repo named getting-started-deploy.md (dash vs underscore)

Reply:
Fixed link, thanks!